evalml/tests/data_checks_tests/test_class_imbalance_data_check.py

```diff
@@ -236,7 +236,7 @@ def test_class_imbalance_nonnumeric(input_type):
     X = pd.DataFrame()
     y_bools = pd.Series([True, False, False, False, False])
     y_binary = pd.Series(["yes", "no", "yes", "yes", "yes"])
-    y_multiclass = pd.Series(["red", "green", "red", "red", "blue", "green", "red", "blue", "green", "red"])
+    y_multiclass = pd.Series(["red", "green", "red", "red", "blue", "green", "red", "blue", "green", "red", "red", "red"])
```

Comment:
increase `red` class to maintain the class imbalance data checks

Reply:
Oh I see, so because we are computing "majority vs current" ratio for each class, instead of a hard 10% threshold as before, we need more "red" in order for "green" and "blue" to trigger the error below. Yes?